src/fees/hooks.ts

```diff
@@ -59,6 +59,7 @@ export function useMaxSendAmount(
     if (!shouldRefresh) return
     const feeEstimate = feeEstimates[tokenAddress]?.[feeType]
     if (
+      (feeType === FeeType.SWAP && balance.gt(0)) ||
```

Comment:
don't dispatch the fee estimation here in case the user has selected a token and the previously entered amount exceeds the balance. the fee estimation saga automatically displays a red error banner for insufficient balance